tests/config/rsc/user.yaml

```diff
@@ -1,6 +1,7 @@
 levels:
   custom:
-    inherits_from: "threshold"
+    inherits_from:
```

Comment:
what about testing inheriting from levels in both the base config file and the custom file?

Reply:
that tests for multiple levels from one file, but not one level from the custom config and one level from the base statick config.

Reply:
[b7ee16e](https://github.com/sscpac/statick/pull/405/commits/b7ee16ec5c1d6e31a0c4f2bda83656cfac839111). This test uses a base configuration file and a user configuration file. The level we are testing inherits from levels in both base and user configuration files. I had to make the test `config.yaml` have reporting plugins to test that part, where the addition matches the main Statick configuration file.